docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,375 @@
+# How to use go-plugin-runner in APISIX Ingress
```

Comment:
Shouldn't we follow this format:

```yaml
---
title: Title
keywords:
  - APISIX
  - Ingress
  - ...
  - ...
description: This document contains information about ...
---
```

Reply:
Thank you very much for your suggestion; I will revise it carefully.

Reply:
Solved～